File: src/router/Models/Order.cs
namespace ServerlessPizzaAPI.Models
{
    using System;
    using System.Collections.Generic;

    public class Order
    {
        public Order(string id, string name, DateTime placed, IEnumerable<Event> events, Details details)
        {
            Id = id;
            Name = name;
            Placed = placed;
            Events = events;
            Details = details;
        }

        public string Id { get; }
        public string Name { get; }
        public DateTime Placed { get; }
        public IEnumerable<Event> Events { get; }
        public Details Details { get; }
    }
}

File: src/delivery/model/order.go
package model

type Order struct {
	ID      string  `dynamodbav:"id"`
	Name    string  `dynamodbav:"name"`
	Placed  string  `dynamodbav:"placed"`
	Events  []Event `dynamodbav:"events"`
	Details Details `dynamodbav:"details"`
}
